root_orchestrator/system-manager-python/blueprints/clusters_blueprints.py

```diff
@@ -22,6 +22,7 @@
         "virtualization": {"type": "array", "items": {"type": "string"}},
         "more": {"type": "object"},
         "worker_groups": {"type": "string"},
+        "addons_supported": {"type": "array", "items": {"type": "string"}},
```

Comment:
We can call it something else, I am not adamant about this name.

Reply:
I think this name is good, I simply swapped it a bit to "supported_addons"